homeassistant/components/template/alarm_control_panel.py

```diff
@@ -51,15 +43,15 @@
 
 _LOGGER = logging.getLogger(__name__)
 _VALID_STATES = [
```

Comment:
Side note: This could be a set.

Reply:
Both this and below remark to handle in follow-up PR